platform-sdk/sdk/settings.txt

```diff
@@ -47,6 +45,7 @@ state.saveReconnectStateToDisk,           false
                                           #############################
 
 metrics.csvFileName,                      PlatformTesting
+prometheus.enabled,                       false

```

Comment:
Why is that false? I do not see any valid point why prometheus should not be active by default.

Reply:
Intention was to leave it off since I didn't think anybody was using it when running locally with this settings file. I will allow it to be enabled if you think we may be useful (or at the very least, not harmful).